rules.txt

```diff
@@ -329,17 +329,28 @@
 /do-?something-?big                       301  /2024-02-27-do-something-big-explore-a-career-with-the-city-of-philadelphia/
 /dobbins                                  301  /programs/community-schools/murrell-dobbins-cte-high-school/
 /documents/2015-income-based-wage-tax-refund-petition 301 /documents/income-based-wage-tax-refund-petition/
+/documents/2021-(.*)                      301  /departments/department-of-revenue/forms-documents/tax-forms-and-instructions/#/
+/documents/2022-(.*)                      301  /departments/department-of-revenue/forms-documents/tax-forms-and-instructions/#/
```

Comment:
I'm seeing a lot more redirects in the spreadsheet than are on this PR, do we have them all captured somewhere? https://phila-my.sharepoint.com/:x:/g/personal/briana_morgan_phila_gov/EYlkKcswv9JAtR7Wi268M7wBqXxGbmW3v5f5VZ13eUgifA?e=XE3tgl

Reply:
sorry about that, the spreadsheet opened on the last column and I didn't see the other ones. I'll add in the others